sphinx/util/texescape.py

```diff
@@ -49,6 +43,22 @@
     # OHM SIGN U+2126 is handled by LaTeX textcomp package
 ]
 
+# A map to avoid TeX ligatures or character replacements in PDF output
+# xelatex/lualatex/uplatex are handled differently (#5790, #6888)
+ascii_tex_replacements = [
+    # Note: the " renders curly in OT1 encoding but straight in T1, T2A, LY1...
+    #       escaping it to \textquotedbl would break documents using OT1
+    #       Sphinx does \shorthandoff{"} to avoid problems with some languages
+    # Can't replace - by {}- because it breaks URLs
+    # ('-', '{}-'),  # -- and --- are TeX ligatures
+    (',', '{},'),  # ,, is a TeX ligature in T1 encoding
+    # the next two require textcomp package
+    ("'", r'\textquotesingle{}'),  # else ' renders curly, and '' is a ligature
+    ('`', r'\textasciigrave{}'),   # else \` and \`\` render curly
```

Comment:
It will be possible to allow line-breaks in parsed literals at the left and right ticks by updating `\sphinxbreaksattexescapedchars` in sphinx.sty but this would be easier if escaping here is `\sphinxquotesingle` and `\sphinxasciigrave`. 

Actually all escaping should be to `\sphinx...` prefixed macros for flexibility in obtaining special effects simply by redefining them.